src/utils/cssStyles.ts

```diff
@@ -12,7 +12,7 @@ type Props = {
 	endColor ?: string,
 }
 
-export function bgBlur(props: Props) {
```

Comment:
just recommend, `bgBlur(props?: Props): object` is the same as 2 times Type Assertion